.golangci.yml

```diff
@@ -0,0 +1,11 @@
+run:
+  timeout: 30s
+  issues-exit-code: 1
+
+linters:
+  disable:
+    # TODO remove these linters and fix the deadcode/errcheck/unused/varcheck
+    - deadcode
+    - errcheck
+    - unused
+    - varcheck
```

Comment:
we can remove `deadcode`, `varcheck` and `unused` from the list

Reply:
also missing new line at the end of the file

Reply:
and regarding errcheck, given the fact that you fixed most of then we can enable errcheck and disable it only for terminal functions like `Println`

Reply:
(see my other configuration suggestion)